manual/manual.adoc

```diff
@@ -349,21 +350,25 @@ You may also encounter any one of these special items while exploring:
 [cols="1,2",width="90%",align="center",valign="middle"]
 |==========================
 | **Keys** +
-image:../sprites/bkeya0.png[Blue Key]
-image:../sprites/rkeya0.png[Red Key]
-image:../sprites/ykeya0.png[Yellow Key] +
-image:../sprites/bskua0.png[Blue Key]
-image:../sprites/rskua0.png[Red Key]
-image:../sprites/yskua0.png[Yellow Key] |
+image:../sprites/bkeya0.png[Blue Passcard]
+image:../sprites/ykeya0.png[Yellow Passcard]
+image:../sprites/rkeya0.png[Red Passcard] +
+image:../sprites/bskua0.png[Blue Skeleton Key]
+image:../sprites/yskua0.png[Yellow Skeleton Key]
+image:../sprites/rskua0.png[Red Skeleton Key] |
 Allow you to open certain locked doors and activate locked switches.
 Usually essential to be able to progress, although they sometimes allow
 access to secret areas.
+
+If you can't see the colors, check the center of the card or the horns
+on the skull: +
+• the first two letters of "Azure" have lots of diagonal lines. +
```

Comment:
I'm so confused. Azure? The planet horizon?

Reply:
TIL in the absolut enshittification of everything: if you look up "azure" in DuckDuckGo, you get absolutel no results whatsoever for the colour, it's *all* goddamn Microsoft.

(This mnemonic actually started out in the Spanish version where there would be no expectation that someone would have to look up the word.)

And I just remembered that the entire line "Horizon is a whole new world" isn't even in the E3 endtext anymore.

Fine I'll get rid of it.